source/_integrations/zha.markdown

```diff
@@ -575,58 +575,61 @@ After restoring a Home Assistant backup, you can reconfigure ZHA or migrate to a
 
 Manual backups can also be created from the configuration page under **Network Settings**.
 
-### Migrating to a new Zigbee Coordinator adapter inside ZHA
+### Migrating to a new Zigbee adapter inside ZHA
 
-ZHA supports migrating the Zigbee network between different Zigbee Coordinators based on chips from Silicon Labs, Texas Instruments, or ConBee/RaspBee if the backup was made from inside ZHA.
+ZHA supports migrating the Zigbee network between different Zigbee adapters based on chips from Silicon Labs, Texas Instruments, or ConBee/RaspBee if the backup was made from inside ZHA.
 
 #### Prerequisites
 
 Confirm you meet the following requirements before migrating:
 
-- The previous coordinator is used in the ZHA {% term integration %} and _not_ in deCONZ or MQTT.
+- The previous adapter is used in the ZHA {% term integration %} and _not_ in deCONZ or MQTT.
 - The radio type is one of the following:
   - ezsp _(Silicon Labs EmberZnet)_
   - znp _(Texas Instruments Z-Stack ZNP)_
   - deCONZ _(ConBee/RaspBee from dresden elektronik)_
     - Be sure it is running [firmware 0x26700700 (from 2021-08-18)](https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Firmware-Changelog) or later.
 
-{% details "To migrate to a new Zigbee coordinator radio inside ZHA:" %}
+{% details "To migrate to a new Zigbee adapter inside ZHA:" %}
 
 {% important %}
-You will not be able to control your existing Zigbee devices until the new coordinator fully joins the network after the migration. **This can take a few minutes.**
+You will not be able to control your existing Zigbee devices until the new adapter fully joins the network after the migration. **This can take a few minutes.**
```

Comment:
note: This is not anything new, but technically a bit inaccurate, since the coordinator hosts the entire network itself. Only all other devices need to re-join "the same network" (since we migrated it).